sdk/loadtestservice/load-testing-rest/samples-dev/sample.ts

```diff
@@ -47,28 +49,18 @@ async function main() {
     throw new Error("Test ID returned as undefined.");
 
   // Uploading .jmx file to a test
-  const fileUploadResult = await client
-    .path("/loadtests/{testId}/files/{fileId}", testCreationResult.body.testId, fileId)
-    .put({
-      contentType: "multipart/form-data",
-      body: {
-        file: readStream,
-      },
-    });
+  const fileUploadResult = await (await beginUploadAndFileValidation(client, testId, "sample", readStream)).pollUntilDone();
 
-  if (isUnexpected(fileUploadResult)) {
-    throw fileUploadResult.body.error;
-  }
+  if (isUnexpected(fileUploadResult)) throw new Error("There is some issue in validation, please make sure uploaded file is a valid JMX.");
```

Comment:
This should check if `poller.getOperationState().status` is not `succeeded`

Reply:
Updated.